### PR TITLE
fix(file-management): ensure absolute path for notebook and requirements files are used in the entire application without getting affected by directory change

### PR DESCRIPTION
**Pull request by priyakanabar-crest**
This PR addresses a critical bug where changing the working directory breaks file management, including saving and publishing notebooks. The solution ensures that the application works consistently

### CODE REVIEW

File: zt_cli/cli.py [hunk 1]
```diff
@@ -205,7 +210,7 @@ def notebook(
     """
     Start the Zero-True application.
     """
-
+    set_zt_path
```
Comment (Carson-Shaar) [inline zt_cli/cli.py:213]: Parentheses missing

File: zt_cli/cli.py [hunk 2]
```diff
@@ -18,6 +18,11 @@
 cli_app = typer.Typer()
 
 
+# Set ZT_PATH environment variable to the current working directory
+def set_zt_path():
+    if "ZT_PATH" not in os.environ:
+        os.environ["ZT_PATH"] = os.getcwd()
+
```
Comment (Carson-Shaar) [inline zt_cli/cli.py:25]: Always set even if env variable already exists

Comment (Carson-Shaar) [reply zt_cli/cli.py:25]: May need to use os.normpath for compatibility on all operating systems

File: zt_dev_cli/cli.py [hunk 3]
```diff
@@ -13,6 +13,12 @@
 cli_app = typer.Typer()
```
[the remark this answers is not in the document]
Comment (priyakanabar-crest) [reply zt_dev_cli/cli.py:14]: Will update that

File: zt_cli/cli.py (hunk 1 above)
Comment (priyakanabar-crest) [reply zt_cli/cli.py:213]: sorry, for not checking properly will update this

**Comment by priyakanabar-crest** [comment]
@Carson-Shaar I have updated the PR

**Pull request closed without merging**